verilog/analysis/checkers/proper_parameter_declaration_rule.cc

```diff
@@ -79,11 +122,24 @@ void ProperParameterDeclarationRule::HandleSymbol(
       } else if (ContextIsInsideModule(context) &&
                  !ContextIsInsideFormalParameterList(context)) {
         violations_.insert(LintViolation(symbol, kParameterMessage, context));
+      } else if (ContextIsInsidePackage(context)) {
+        if (!package_allow_parameter_) {
+          violations_.insert(LintViolation(symbol, kParameterMessage, context));
+        }
```

Comment:
We have a opportunity here to provide an autofix to change `parameter` to `localparam`.
That way, in the language server it is just one click away to get fixed.

Might be a good follow-up change.